src/platforms/mirserver/miropenglcontext.cpp

```diff
@@ -191,11 +193,17 @@ void MirOpenGLContext::doneCurrent()
 #if QT_VERSION < QT_VERSION_CHECK(5, 7, 0)
 QFunctionPointer MirOpenGLContext::getProcAddress(const QByteArray &procName)
 {
-    return eglGetProcAddress(procName.constData());
+    QFunctionPointer p = (QFunctionPointer) dlsym(RTLD_DEFAULT, procName.constData());
+    if (!p)
+	p = eglGetProcAddress(procName.constData());
+    return p;
```

Comment:
It is best to still include braces for single statement `if` blocks. However, this could also be changed to `return p ? p : eglGetProcAddress(procName.constData());` instead, I think.